plugins/modules/source_control/github/github_deploy_key.py

```diff
@@ -128,206 +125,141 @@
     password: "supersecretpassword"
     otp: 123456
 
-# add a read-only deploy key to a repository hosted on GitHub Enterprise
+# Create a deploy key in a GitHub enterprise repository
 - github_deploy_key:
-    github_url: "https://api.example.com"
-    owner: "janedoe"
-    repo: "example"
+    owner: "johndoe"
```

Comment:
Why replace `janedoe` with `johndoe`?

Reply:
ha, no idea. I'll put it back

Reply:
ah, no. It's because all the previous examples use johndoe, OCD kicked in and automatically changed this. I'm still putting `janedoe` back! :)